tox.ini

```diff
@@ -4,10 +4,11 @@ envlist = py34, py33, py32, py27, py27-configparser, pypy
 [testenv]
 passenv = HOME
 deps=
+  distribute
   pytest
   mock
 commands=
-  py.test [] tests.py
+  py.test [] tests.py tests
```

Comment:
We should probably move tests.py to the tests folder (and maybe call them end-to-end or cli tests or something) instead.